esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +214,36 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    def test_node_list_long(self):
+        """ Tests functionality of "esi node list" if using node_uuid.
+            returns the list of nodeif optional argument is not passed.
```

Comment:
I'm not sure what this part of the comment means, since the test shouldn't return anything (was this referring to what gets returned in the exception that gets thrown?)

Reply:
Yes, But let me change the wording of it to be more clear.